src/components/balance/BalancePlasm.vue

```diff
@@ -35,6 +32,13 @@
     <ModalTransferAmount
       v-if="modalTransferAmount"
       v-model:isOpen="modalTransferAmount"
+      v-on:completeTransfer="completeTransfer"
+      :all-accounts="allAccounts"
```

Comment:
The UI will not make any distinctions even if there are no addresses loaded to the page. It might be better to disable the transfer button or some indication to let the user know that they don't have any loaded accounts

Reply:
I changed that if the account is not loaded, the transfer button is disabled.